api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -41,16 +43,30 @@ public ApiClient fireCloudApiClient(UserAuthentication userAuthentication,
     return apiClient;
   }
 
+  private String getAccessToken(WorkbenchEnvironment workbenchEnvironment) throws IOException {
```

Comment:
nit: I'd rename to something more explicit, e.g. `getWorkbenchServiceAccessToken`; or something to disambiguate this from EUCs

Reply:
Done.